libraries/AP_IOMCU/AP_IOMCU.cpp

```diff
@@ -786,7 +801,7 @@ void AP_IOMCU::set_safety_mask(uint16_t chmask)
  */
 bool AP_IOMCU::healthy(void)
 {
-    return crc_is_ok && protocol_fail_count == 0;
+    return crc_is_ok && stats.protocol_fail_count > 5;
```

Comment:
I wonder if this should be split into two PRs...

Reply:
Feel free to cherry pick this part into a PR. It is needed to fix a 0.6% chance of failing to arm on QuadPlane (when everything is otherwise normal). I put it in this, because I didn't think this would be stalled on lots of changes to IOMCU :)